content/kubernetes/active-active/preview/global-config.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: Set global database configurations
+linkTitle: Global configuration
+description: The REAADB contains the field '.spec.globalConfigurations' and through this the database configurations are set.
+weight: 50
+alwaysopen: false
+categories: ["Platforms"]
+aliases: {
+    /kubernetes/active-active/preview/global-config/,
+
+}
+---
+
+{{<banner-article bannerColor="#fff8dc">}}
+This feature is currently in public preview. Contact Redis support if you plan to use this feature.
+See [Create Active-Active databases for Kubernetes]({{<relref "/kubernetes/active-active/create-aa-database.md">}}) for the currently supported procedure.
+{{</banner-article>}}
+
+The Redis Enterprise Active-Active database (REAADB) custom resource contains the field '.spec.globalConfigurations'. This field sets configurations for the Active-Active database across all participating clusters, such as memory size or shard count.
+
+The [REAADB API reference](https://github.com/RedisLabs/redis-enterprise-operator/blob/master/deploy/redis_enterprise_active_active_database_api.md) contains a full list of available fields.
+
+## Edit global configurations
+
+1. Edit or patch the REAADB custom resource with your global configuration changes.
+
+  The example command below patches the reaadb custom resource to set the global memory size to 200MB:
+
+  ```sh
+  kubectl patch reaadb example-aadb-1 --type merge --patch /
+  '{"spec": {"globalConfigurations": {"memorySize": "200mb"}}}'
+  ```
+
+1. Verify the status is `active` and the spec status is `Valid`.
+
+  ```sh
+  kubectl get reaadb example-aadb-1
+
+  NAME             STATUS   SPEC STATUS   GLOBAL CONFIGURATIONS REDB   LINKED REDBS
+  example-aadb-1   active   Valid    
+  ```
+
+1. View the global configurations on each participating cluster to verify they are synced.
+
+  ```sh
+  kubectl get reaadb <REAADB name> -o yaml
+  ```
+
+## Set global database secret
+
+One of the fields available for `globalConfigurations` is `databaseSecretName` which can point to a secret containing the database password.
+
+### On an existing participating cluster, do the following
```

Comment:
I think instead of having headings - you might want to change these to sentences? The headings seem a little long